config/src/main/java/com/typesafe/config/impl/ConfigDocumentParser.java

```diff
@@ -72,9 +74,17 @@ private Token nextToken() {
         private Token nextTokenIgnoringWhitespace(Collection<AbstractConfigNode> nodes) {
```

Comment:
let's rename this `nextTokenCollectingWhitespace` maybe - would make more sense given its current behavior.